spec/models/types_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Types do
+  describe 'constant FederalPovertyLevels' do
+    def fetch_fpl_data(year)
+      ::Types::FederalPovertyLevels.detect do |fpl_hash|
+        fpl_hash[:medicaid_year] == year
+      end
+    end
+
+    context 'for year 2022' do
+      before do
```

Comment:
what are you testing with these specs? it looks like the spec tests how Ruby works

Reply:
**2021** FPL values will be used for **2022** applications and **2022** FPL values will be used for **2023** applications.

Reply:
understood, that answers why we're keeping these values

now we need to understand what exactly you're testing--it appears that you're testing how Ruby works, the JavaScript equivalent would be:

``` js
const obj = { foo: 'bar' }

expect obj.foo === 'bar'
```

this doesn't do anything meaningful